package.json

```diff
@@ -14,6 +14,8 @@
     "mongo_queue_processor": "node extensions/mongoProcessor/mongoProcessorTask.js",
     "garbage_collector": "node extensions/gc/service.js",
     "test": "mocha --recursive tests/unit",
```

Comment:
I think it could also be worthwhile for development to reintroduce the global `ft_test` command to run all functional tests. i.e.

`"ft_test": "mocha --recursieve $(find tests/functional -name '*.js')",`